scripts/createYOEOLabelsFromTORSO-21.py

```diff
@@ -12,7 +12,7 @@
 CLASSES = {
     'bb_classes': ['ball', 'goalpost', 'robot'],
     'segmentation_classes': ['field edge', 'lines'],
-    'ignored_classes': ['obstacle', 'L-Intersection', 'X-Intersection', 'T-Intersection']
+    'skip_classes': ['obstacle', 'L-Intersection', 'X-Intersection', 'T-Intersection']
```

Comment:
What is the purpose of the skip classes anyway? Why do they exist if we explicitly list the positive cases.

Reply:
Ah okay, maybe it is nice as a sanity check. If we encounter unknown classes.